### ISSUE TITLE
References shazam S3 log bucket

### ISSUE THREAD

**Issue by andreweacott** [shazam/scala-datapipeline-dsl#2]
ResourceJsonRenderer includes a reference to a Shazam-specific S3 logs bucket which can't be overridden :
```
val cluster = IdTypeAndNameJsonRenderer(emrCluster.objectType, emrCluster).merge {
        ("enableDebugging" -> enableDebugging.map(_.toString)) ~
          ("releaseLabel" -> releaseLabel) ~
          ("masterInstanceType" -> masterInstanceType) ~
          ("coreInstanceCount" -> coreInstanceCount.map(_.toString)) ~
          ("coreInstanceType" -> coreInstanceType) ~
          ("coreInstanceBidPrice" -> coreInstanceBidPrice.map(_.toString)) ~
          ("useOnDemandOnLastAttempt" -> useOnDemandOnLastAttempt.map(_.toString)) ~
          ("terminateAfter" -> terminateAfter.map(_.toString)) ~
          ("emrLogUri" -> s"s3://szm-den-aws-data-pipeline/temp/$${environment}/$pipelineName/logs/emr") ~
          ("keyPair" -> keyPair) ~
          ("region" -> region) ~
          ("applications" -> applications) ~
          ("configuration" -> configuration.map { conf => "ref" -> conf.name.replaceAllLiterally(" ", "-") })
```

**Comment by andreweacott** [comment]
Have submitted PR #1 with fix

**Comment by markhatton** [comment]
Closed as per #1
